geomstats/geometry/special_orthogonal.py

```diff
@@ -1278,246 +1139,219 @@ def tait_bryan_angles_from_rotation_vector(
-
         random_point = gs.random.rand(n_samples, self.dim) * 2 - 1
-        random_point = self.regularize(random_point, point_type='vector')
-        if point_type == 'matrix':
-            random_point = self.matrix_from_rotation_vector(random_point)
+        random_point = self.regularize(random_point)
 
         if n_samples == 1:
             random_point = gs.squeeze(random_point, axis=0)
 
         return random_point
 
-    @geomstats.vectorization.decorator(['else', 'point', 'point_type'])
+    @geomstats.vectorization.decorator(['else', 'vector', 'point_type'])
     def exp_from_identity(self, tangent_vec, point_type=None):
         """Compute the group exponential of the tangent vector at the identity.
 
+        As rotations are represented by their rotation vector,
+        which corresponds to the element `X` in the Lie Algebra such that
+        `exp(X) = R`, this methods returns its input without change.
+
         Parameters
         ----------
-        tangent_vec : array-like, shape=[n_samples, {dim, [n, n]}]
+        tangent_vec : array-like, shape=[n_samples, 3]
         point_type : str, {'vector', 'matrix'}, optional
             default: self.default_point_type
 
         Returns
         -------
-        point : array-like, shape=[n_samples, {dim, [n, n]}]
+        point : array-like, shape=[n_samples, 3]
         """
-        if point_type == 'matrix' and self.n > 3:
-            return gs.linalg.expm(tangent_vec)
-        if point_type == 'matrix':
-            tangent_vec = self.vector_from_skew_matrix(tangent_vec)
-            return self.matrix_from_rotation_vector(tangent_vec)
-
         return tangent_vec
 
-    @geomstats.vectorization.decorator(['else', 'point', 'point_type'])
+    @geomstats.vectorization.decorator(['else', 'vector', 'point_type'])
     def log_from_identity(self, point, point_type=None):
         """Compute the group logarithm of the point at the identity.
 
+        As rotations are represented by their rotation vector,
+        which corresponds to the element `X` in the Lie Algebra such that
+        `exp(X) = R`, this methods returns its input after regularization.
+
+
         Parameters
         ----------
-        point : array-like, shape=[n_samples, {dim, [n, n]}]
-        point_type : str, {'vector', 'matrix'}, optional
-            default: self.default_point_type
+        point : array-like, shape=[n_samples, 3]
 
         Returns
         -------
-        tangent_vec : array-like, shape=[n_samples, {dim, [n, n]}]
+        tangent_vec : array-like, shape=[n_samples, {dimension, [n, n]}]
         """
-        if point_type == 'vector':
-            tangent_vec = self.regularize(point, point_type=point_type)
-        elif point_type == 'matrix':
-            point = self.rotation_vector_from_matrix(point)
-            tangent_vec = self.skew_matrix_from_vector(point)
-        return tangent_vec
+        return self.regularize(point)
+
+    def lie_bracket(
```

Comment:
Same comment: if we keep the class LieAlgebra, then we should create the lie algebra of SO and SE and put these methods there. Alternatively, we could delete the `LieAlgebra` class for the moment, and keep all of these methods in the groups.

Reply:
yes although it's not really the same bracket: in lie algebra it's the usual commutator, but in Lie group it's the lie bracket for left-invariant vector fields

Reply:
But the space of left-invariant vector fields is isomorphic to the Lie algebra, and therefore can be identified to it, and to any tangent space.

As a matter of fact, this lie_bracket function operates on tangent_vectors, and not tangent vector fields, so it amounts to the lie bracket on the lie algebra when `base_point=None`?

Reply:
yes but the question is, is this function useful when base_point not equal to None? Should that function (with possible base_point) be in LieAlgebra? This corresponds to the question, does the LieAlgebra represent TeG or the set of Left-inv vect fields? Although isomorphic, they are not represented the same way, and having base_point argument implies the LieAlgebra is not only TeG but rather L(G)